overlord/devicestate/devicestate_test.go

```diff
@@ -71,7 +72,10 @@ type deviceMgrSuite struct {
 	restoreOnClassic func()
 }
 
-var _ = Suite(&deviceMgrSuite{})
+var (
+	_             = Suite(&deviceMgrSuite{})
```

Comment:
This reads slightly odd. Having them in their own _var_ lines might be nicer in this particular case.